torchgeo/models/resnet.py

```diff
@@ -37,6 +38,9 @@
     K.CenterCrop(224),
     K.Normalize(mean=_min, std=_max - _min),
     K.Normalize(mean=torch.tensor(0), std=1 / torch.tensor(255)),
+    Lambda(lambda x: torch.clamp(x, min=0.0, max=255.0)),
+    Lambda(lambda x: x.to(torch.uint8).to(torch.float)),  # type: ignore[no-any-return]
+    K.Normalize(mean=torch.tensor(0), std=torch.tensor(255)),
```

Comment:
This is actually the exact opposite of what we want. This would divide by 255, not multiply. The above normalization is the correct one.

Reply:
`transforms.ToTensor(),` right before `imagenet_normalization()`

Reply:
Good catch. In that case, I would remove all 4 lines, so we neither multiply by 255 nor divide by 255. This will make the transform faster.